config/cli/fleet-eth.test.json

```diff
@@ -1,5 +1,6 @@
 {
   "LogLevel": "DEBUG",
+  "NoDiscovery": false,
```

Comment:
Not related: I wonder why this config misses `Rendezvous: true`.

Reply:
No idea. I'll add it.